apps/sms/views/conversation/js/conversation.js

```diff
@@ -815,8 +816,9 @@ var ConversationView = {
     // Update the stored thread data
     Threads.registerMessage(message);
 
-    this.appendMessage(message);
-    TimeHeaders.updateAll('header[data-time-update]');
+    return this.appendMessage(message).then(
+      () => TimeHeaders.updateAll('header[data-time-update]')
```

Comment:
I'm not sure this call is actually needed... but I'm not really sure where it's done otherwise